selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java

```diff
@@ -42,14 +40,32 @@ public void goToWiki() {
     $(ELEMENT_WIKI_LINK_PLF).click();
     refreshUntil($(ELEMENT_WIKI_LINK_PLF), Condition.visible, 500);
   }
-
+  /**
+   * Go to People portlet
+   */
+  public void goToPeople() {
+    info("--Go to People--");
+    refreshUntil($(ELEMENT_PEOPLE_LINK_PLF), Condition.visible, 500);
+    $(ELEMENT_PEOPLE_LINK_PLF).waitUntil(Condition.appears, Configuration.timeout);
+    $(ELEMENT_PEOPLE_LINK_PLF).click();
+    refreshUntil($(ELEMENT_PEOPLE_LINK_PLF), Condition.visible, 500);
+  }
+  /**
+   * Go to Chat
+   */
   public void goToChat() {
     info("--Go to chat--");
     refreshUntil($(byClassName("status-dropdown")), Condition.visible, 1000);
     $(byClassName("status-dropdown")).click();
     $(byClassName("notif-chat-open-link")).waitUntil(Condition.appears, Configuration.timeout).click();
   }
 
+  public void backToHomeFromChat() {
+    info("--Go to Home--");
+    refreshUntil($(byClassName("home-button")), Condition.visible, 1000);
+    $(byXpath("//div[@data-original-title='Back to Home']/a/i")).waitUntil(Condition.visible,Configuration.timeout).click();
```

Comment:
Not sure it is a good practice to rely on a text ("Back to Home"). It can be changed by translators, so it is fragile. Can't we use an id or a class ?
Same remark for others selectors using text.

Reply:
Yes true, Fixed done.

Reply:
It should be fixed in other places where we also use labels.

Reply:
Ok